java/java-jersey2-alternatives/src/main/webapp/WEB-INF/web.xml

```diff
@@ -88,13 +88,12 @@
     <servlet-name>OpenApi</servlet-name>
     <url-pattern>/openapi/*</url-pattern>
   </servlet-mapping>
-<!-- alternatively use own Bootstrap servlet
+
```

Comment:
why is this uncommented

Reply:
Fixed (commented again), doing some tests with the Bootstrap class.